### DISCUSSION TITLE
PWM control with MEGABAS

### DISCUSSION THREAD

**Discussion by rmwilder** [tagyoureit/nodejs-poolController#1006] [REM]
Is it possible to control a 12v, 4 wire, PWM fan with the MEGABAS? I haven’t done much digging around yet but I figured I’d ask. RPM feedback would be nice to monitor but not crucial.

**Comment by jaysalazar** [comment]
Did you ever make any progress on this?

**Reply by tagyoureit** [reply]
Nope.  Haven't looked and this and, unfortunately, don't have a lot of bandwidth currently.
